account_payment_term_cutoff_day/models/account_payment_term_line.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2025 Quartile
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from dateutil.relativedelta import relativedelta
+
+from odoo import fields, models
+
+
+class AccountPaymentTermLine(models.Model):
+    _inherit = "account.payment.term.line"
+
+    cutoff_day = fields.Integer(
+        help="Specify the cutoff day of the month for adjusting invoice due dates."
+        "For example, if you set this field to 20, any invoice dated on the 21st or later "
+        "will have its due date moved to the following month.",
```

Comment:
To avoid unnecessary confusion. What do you think?
```suggestion
        default=31,
        help="Specify the cutoff day of the month for adjusting invoice due dates."
        "For example, if you set this field to 20, any invoice dated on the 21st or later "
        "will have its due date moved to the following month."
        "Setting 0 is treated the same as having no cutoff day, which in practice is the "
        "same as setting 31.",
```